ssl/bio_ssl.c

```diff
@@ -93,6 +93,7 @@ static int ssl_read(BIO *b, char *out, int outl)
     SSL *ssl;
     int retry_reason = 0;
     int r = 0;
+    size_t read;
```

Comment:
Because read is the name of a function, which is probably why you got that warning.